examples/hashing/src/main.sw

```diff
@@ -73,36 +92,57 @@ fn main() {
     log(sha_hashed_enum);
     log(sha_hashed_struct);
 
+
```

Comment:
What is the reason for the extract newlines here and in other places?

Reply:
and does this mean that when I rerun the formatter another new line will be added?

Reply:
@mohammadfawaz I just noticed this issue https://github.com/FuelLabs/sway/issues/2691 - @eureka-cpu possibly related?

Reply:
@mohammadfawaz & @mitchmindtree yes, there's an issue currently where newlines will continue to get added if there is a comment with a newline in its span. Doesn't seem to happen for comments that don't have an extra newline or in places without comments.